packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -208,11 +237,13 @@ class Table extends Component {
   /**
    * @property {Object} tableData
    */
-  @computed('rawData', 'columns', 'options.showTotals.{grandTotal,subtotal}')
+  @computed('rawData', 'columns', 'options.showTotals.{grandTotal,subtotal}', 'request')
   get tableData() {
     let tableData = this._computeSubtotals(),
```

Comment:
Does this feature play nicely with the client side implementation?

Reply:
and line 220/251

Reply:
Somewhat nicely. Client side will get both server side and client side rows. The order of the server side rows just gets pushed to the bottom.